CHANGELOG.md

```diff
@@ -26,6 +26,25 @@ so make sure you follow the template
 
 -->
 
+## 2.0.0.dev0 2021-xx-xx
+
+### Added
+
+* Add support for django 3.2
+* `auth/backends` now works without `authtools`
+    * `ProfileModelBackend` will not be available if `authtools` is not able to be imported (`authtools` does not currently work with django 3.2)
+* ````
+* Add `add_autoreload_extra_files()`
+* Add ability for checks to ignore apps/models using a regex instead of just a static string 
+
+  
+### Fixed
+* `GenericUserProfile.normalize_email` can now be overridden on child classes and will work as expected
+
+### Breaking Changes
+* Drop support for django 2.2
```

Comment:
if we're dropping support for django22 why is that still being tested? 
```
-        python-version: ['36', '37', '38']
-        django-version: ['22']
+        python-version: ['36', '37', '38', '39']
+        django-version: ['22', '32']
```

Reply:
Whoops that should say 1.11